SharedSource/Core/Utilities/MemberInfoEqualityComparer.cs

```diff
@@ -47,7 +48,7 @@ sealed partial class MemberInfoEqualityComparer<T> : IEqualityComparer<T> where
     /// The idea for this method, but not the code, was taken from http://blogs.msdn.com/b/kingces/archive/2005/08/17/452774.aspx.
     /// </para>
     /// </returns>
-    public bool Equals (T one, T two)
+    public bool Equals ([AllowNull] T one, [AllowNull] T two)

```

Comment:
that doesnt work because of the constraint on `T: MemberInfo`, as `T?` would violate nullability of MemberInfo.
As `T: MemberInfo` implies `T: class` however `bool Equals(T? one, T? two)` is possible

Reply:
You're right, must have overlooked this. `bool Equals(T? one, T? two)` it is.